src/gpodder/gtkui/main.py

```diff
@@ -2638,6 +2652,10 @@ def update_progress(channel):
 
                 util.idle_add(update_progress, channel)
 
+            if nr_update_errors > 0:
+                self.notification(_('%d channel(s) failed to update, view all episodes to see them') % nr_update_errors,
```

Comment:
Maybe we can always show podasts in error? (I've not looked at the code).

Reply:
I can look into it, but there would be no way to hide a channel with temporary errors short of restarting gpodder or successfully updating the channel.

Reply:
Added a patch that always shows the channel and removed the message to view all.